test/suite/test_bug019.py

```diff
@@ -35,15 +35,17 @@
 class test_bug019(wttest.WiredTigerTestCase):
     conn_config = 'log=(enabled,file_max=100K)'
     uri = "table:bug019"
-    entries = 100000
+    entries = 5000
 
     # Modify rows so we write log records. We're writing a lot more than a
     # single log file, so we know the underlying library will churn through
     # log files.
     def populate(self, nentries):
         c = self.session.open_cursor(self.uri, None, None)
         for i in range(0, nentries):
-            c[i] = i
+            # Make the values about 200 bytes - that's about 1MB of data for
+            # 5000 records, so at 10 log files used plus more for overhead.
```

Comment:
I suggest 'so at' -> 'generating'.
I also suggest 'about 200 bytes - that's...' -> 'about 200 bytes. That's...'